content/en/docs/getting-started-guides/fedora/flannel_multi_node_cluster.md

```diff
@@ -1,192 +1,128 @@
 
-## Prerequisites
-
-You need 2 or more machines with Fedora installed.
-
-## Master Setup
-
-**Perform following commands on the Kubernetes master**
-
-* Configure flannel by creating a `flannel-config.json` in your current directory on fed-master. Flannel provides udp and vxlan among other overlay networking backend options. In this guide, we choose kernel based vxlan backend. The contents of the json are:
-
-```json
-{
-    "Network": "18.16.0.0/16",
-    "SubnetLen": 24,
-    "Backend": {
-        "Type": "vxlan",
-        "VNI": 1
-     }
-}
-```
-
-**NOTE:** Choose an IP range that is *NOT* part of the public IP address range.
+To set up a multi-node Kubernetes cluster on Fedora using **kubeadm**, do the following:
 
-Add the configuration to the etcd server on fed-master.
+* **Set up the master**: Follow the [Kubernetes on Fedora - Single-node](https://kubernetes.io/docs/getting-started-guides/fedora/fedora_manual_config/) instructions to set up a Kubernetes Master. After running kubeadm init on the master, copy the **kubeadm join** line from the output. This includes token information needed to connect the node to the master. 
+* **Set up nodes**: Use the instruction below to configure each additional node. 
 
-```shell
-etcdctl set /coreos.com/network/config < flannel-config.json
-```
-
-* Verify that the key exists in the etcd server on fed-master.
-
-```shell
-etcdctl get /coreos.com/network/config
-```
-
-## Node Setup
-
-**Perform following commands on all Kubernetes nodes**
+## Prerequisites
 
-Install the flannel package
+For each node you want to add, follow the Prerequisites and these step from the Setup section of the [Kubernetes on Fedora - Single-node](https://kubernetes.io/docs/getting-started-guides/fedora/fedora_manual_config/) instructions:
 
-```shell
-# dnf -y install flannel
-```
+* Prerequisites
+* Install kubernetes-kubeadm and other software
+* Enable kubelet
 
-Edit the flannel configuration file /etc/sysconfig/flanneld as follows:
+## Set up Nodes
 
-```shell
-# Flanneld configuration options
+1. Join node to the cluster. Using the output from the **kubeadm init** command you ran earlier on the master, run the **kubeadm join** command. The basic structure of the command is as follows:
 
-# etcd url location.  Point this to the server where etcd runs
-FLANNEL_ETCD="http://fed-master:2379"
+    **IMPORTANT**: You may need the **--ignore-preflight-errors=all** option.
```

Comment:
ignoring all preflight errors is not recommended to the end user of kubeadm.
individual errors have to be dealt with.